### ISSUE TITLE
trap support is missing

### ISSUE THREAD

**Issue by solarsea** [troglobit/mini-snmpd#12]
The README states that traps are supported - but there is only a command line option with capital case T that is not handled further in any way.

Is there a plan to support them ?

Regards

**Comment by troglobit** [comment]
You're right, there is no trap support.  The original author (Robert) seems to have had plans in the past to support traps, but it never materialized. Funnily enough I exchanged emails with another mini-snmpd user last week about just this, and he planned to add it.

In the meantime I've removed the misleading information from the README and `getopt()` handling.
